### ISSUE TITLE
Hello, the beer image in wrong place

### ISSUE THREAD

**Issue by Warox23** [VladMosiychuk/Beer2048#1]
The beer image in wrong place
see attached screenshot 
![photo_2018-05-17_14-04-45](https://user-images.githubusercontent.com/13969918/40173684-4feea99e-59db-11e8-839c-05f2cff93fc0.jpg)

**Comment by VladMosiychuk** [comment]
Yep, I knew it. Maybe I'll fix it later. Or u can try doing it yourself.
